app/_locales/en/messages.json

```diff
@@ -1525,6 +1534,9 @@
   "learnMore": {
     "message": "learn more"
   },
+  "learnMoreWithDot": {
+    "message": "Learn more."
```

Comment:
Is this necessary? One with a period and one without?

Reply:
I'm trying to follow the latest content guidelines. We will do a review of all STX content before pushing it to prod and if this is no longer needed, I will remove it.